spec/models/cart_item_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe CartItem, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
If you don't intend to test this model, then remove the spec file.

Reply:
Will do